src/test_all.py

```diff
@@ -127,7 +128,19 @@ def assert_language_name(name):
                     except KeyError:
                         raise Exception(f" * Looks like we have a repeated example here! "
                                         f"Merge outputs before removing duplicates. :-/ \n {instance}")
-
+                        
+                        
+            #Make sure classes are balanced
+            if 'Classification' in data['Categories']:
+                outputs=[ins['output'] for ins in instances]
+                classes = []
+                for i in outputs:
+                    if i not in classes:
+                        classes.append(i)
+                output_int=[classes.index(output) + 1 for output in outputs]
+                assert abs(skew(output_int)) < 0.5, "Classes distribution is skewed"
```

Comment:
I am actually not sure if the `skew` function is the right choice here. 

>  For **unimodal continuous distributions**, a skewness value greater than zero means that there is more weight in the right tail of the distribution.

Reply:
I guess that would be fine. Since we might have negative values as well, I used abs().

Reply:
Consider a simpler metric, like 
- relative popularity of a particular category compared to an average 
- a measure based on entropy might also work (since entropy measures some notion of similarity to uniform distributions)